### ISSUE TITLE
[Forge 1.20.1 Bug]: FTB Library v2001.2.2 doesn't work with the latest version of Teams (and possibly Quests)

### ISSUE THREAD

**Issue by vizthex123** [FTBTeam/FTB-Mods-Issues#1187]
### Mod

FTB Library

### Mod version

2001.2.2

### Forge / Fabric version

Forge 47.2.6

### What issue are you having?

It seems like each mod wats a different version of FTB Library, and I get this error screen when running the latest version of all of them.
![image](https://github.com/FTBTeam/FTB-Mods-Issues/assets/31721920/0475f046-6e23-4f6e-8d4f-647f240fb049)

If I downgrade FTB Library and leave everything else on the latest version, it still doesn't work.
![image](https://github.com/FTBTeam/FTB-Mods-Issues/assets/31721920/000cdd86-1659-4f0a-ae0d-8a34d0f41036)

I have to run Library 2001.1.5 and Quests 2001.3.5, otherwise I get one of the two errors above.

### Steps to reproduce

1. Install FTB Quests 2001.4.4 and Library 2001.2.2. I also have XMod Compat and Teams, but I'm not sure how relevant they are to this issue.
2. Load the game

### Anything else to note?

It'd be nice if the issue tracker on the CF page linked to this one instead of the pull requests page (applies to all FTB mods). I didn't realize there was an issue tracker for these mods for the longest time lol (and having a drop-down selection menu for the "what mod are you having issues with?" part would be nice).

**Comment by NamelessGamer06** [comment]
another user here with additional info, i am having a similar issue with ftb teams and ftb chunks. this issue is only coming up now as the curseforge app only updates on a release ver of a mod by default. basically ftb chunks and ftb teams in my case are limiting the ftb library ver to have to be under a specific ver. now the latest ver of ftb library is over that limit. in my experience, you shouldn't have a upper limit on mod vers unless you know it wont work with that vers and above but this is your mod and do what you want.  info: MC ver 1.20.1, ftb library ver: 2001.2.2, ftb chunks ver: 2001.2.7, ftb teams ver: 2001.2.0.

**Comment by desht** [comment]
> you shouldn't have a upper limit on mod vers unless you know it wont work with that vers and above

Strangely enough, that's _exactly_ why there's an upper limit.

All of the above mods have dependencies, and older versions of FTB Teams will not work with newer FTB Quests or Library versions.  That's how it is.  Updated FTB Quests/Teams/Library/XMod Compat all to the latest released versions and you'll be fine.

**Comment by desht** [comment]
Yeah that's fair.  I'll get Teams 2001.3.0 promoted to release status.

**Comment by vizthex123** [comment]
noice, thanks.

**Comment by desht** [comment]
Done now.
